### ISSUE TITLE
loss function is nan while training

### ISSUE THREAD

**Issue by LocPham263** [microsoft/Recursive-Cascaded-Networks#72]
Hi, I tried multiple options for the number of cascades. For small number of cascades, the training process normally finished at roughly 2e-01 loss. However, for bigger cascades option, the loss function displayed nan value whenever the training process < 1e-01 loss. What should I do now? Thank you in advance.
